appservice/src/intTest/groovy/nu/yona/server/ActivityTest.groovy

```diff
@@ -1579,24 +1582,26 @@ class ActivityTest extends AbstractAppServiceIntegrationTest
 
 		def weekOverviewLastWeek = responseWeekOverviews.responseData._embedded."yona:weekActivityOverviews"[1]
 
-		assertNumberOfReportedDaysForGoalInWeekOverview(weekOverviewLastWeek, timeZoneGoalSocialRichard, 6)
+		assertNumberOfReportedDaysForGoalInWeekOverview(weekOverviewLastWeek, budgetGoalSocialRichard, 6)
 
-		assertDayInWeekOverviewForGoal(weekOverviewLastWeek, timeZoneGoalSocialRichard, expectedValuesRichardLastWeek, "Mon")
+		assertDayInWeekOverviewForGoal(weekOverviewLastWeek, budgetGoalSocialRichard, expectedValuesRichardLastWeek, "Mon")
 
-		assertWeekDetailForGoal(richard, weekOverviewLastWeek, timeZoneGoalSocialRichard, expectedValuesRichardLastWeek)
+		assertWeekDetailForGoal(richard, weekOverviewLastWeek, budgetGoalSocialRichard, expectedValuesRichardLastWeek)
 
 		assertDayOverviewBasics(responseDayOverviewsAll, expectedTotalDays, expectedTotalDays, 14)
-		assertDayOverviewForTimeZoneGoal(responseDayOverviewsAll, timeZoneGoalSocialRichard, expectedValuesRichardLastWeek, 1, "Mon")
+		assertDayOverviewForBudgetGoal(responseDayOverviewsAll, budgetGoalSocialRichard, expectedValuesRichardLastWeek, 1, "Mon")
 
-		assertDayDetail(richard, responseDayOverviewsAll, timeZoneGoalSocialRichard, expectedValuesRichardLastWeek, 1, "Mon")
+		assertDayDetail(richard, responseDayOverviewsAll, budgetGoalSocialRichard, expectedValuesRichardLastWeek, 1, "Mon")
 
 		cleanup:
 		appService.deleteUser(richard)
 
 		where:
-		operatingSystem | expectedValue
-		"ANDROID" | [ : ]
-		"IOS" | [13 : 1]
+		operatingSystem | budget | expectedValue
+		"ANDROID" | 0 | [ 13 : 4]
+		"ANDROID" | 60 | [ : ]
```

Comment:
A comment on why this is empty and not for iOS could help here

Reply:
Pushed an update